### PR TITLE
Addresses Issue #49. 

### PR DESCRIPTION
**Pull request by Balijah**
The image is being pulled from the remote repo and displayed properly but still working on getting the annotations to be pulled and displayed properly.

### CODE REVIEW

**Comment by kate-holdener** [comment]
> The image is being pulled from the remote repo and displayed properly but still working on getting the annotations to be pulled and displayed properly.

Please provide the required information in the PR. Your PR descriptions are very minimal and challenging to interpret.

File: Data_Directory/Images_Directory/.DS_Store
Comment (kate-holdener) [inline Data_Directory/Images_Directory/.DS_Store:1]: Do not include these files in your commits.

**Comment by kate-holdener** [comment]
Your directory structure does not match the main branch.

File: Editor_Directory/index.html [hunk 1]
```diff
@@ -47,42 +47,51 @@
...
+        document.getElementById('bone-name').textContent = boneData.name;
+        document.getElementById('bone-description').textContent = boneData.description;
 
-        document.addEventListener('DOMContentLoaded', async function() {
-            try {
-                const boneData = await fetchJSON(BONE_JSON_PATH);
-                const annotationData = await fetchJSON(ANNOTATIONS_JSON_PATH);
+        // Display bone image
+        const imagePath = 'https://raw.githubusercontent.com/oss-slu/DigitalBonesBox/data/databones/images/femur.jpg' + boneData.image_path.trim();
+        document.getElementById('bone-image').src = imagePath;
```
Comment (kate-holdener) [inline Editor_Directory/index.html:77]: Your image file is hard coded, it should be dynamic, based on the bone id

**Review by jacksayshi — CHANGES_REQUESTED** [review]
Thanks for working on this.
I think your branch is based on your previous sprint branch. Please pull latest commits from origin main of repo (it has the structure we all agreed on for now) and make changes to HTML based on issue requirement. 
Here is the image of current HTML page of our bones. We need corresponding images from data branch to be displayed along with other details.
image

**Pull request merged**